framework/include/utils/MooseTypes.h

```diff
@@ -549,38 +599,33 @@ using ADTemplateVariablePhiGradient =
 using ADVariablePhiGradient = ADTemplateVariablePhiGradient<Real>;
 
 // Templated typed to support is_ad templated classes
-namespace Moose
-{
 template <typename T, bool is_ad>
-using GenericType = typename std::conditional<is_ad, typename ADType<T>::type, T>::type;
-} // namespace Moose
+using GenericType = typename std::conditional<is_ad, typename Moose::ADType<T>::type, T>::type;
```

Comment:
I like to see more namespacing, not less :frowning: Especially for a typename as generic as `GenericType`

Reply:
Yeah, I knew that would be controversial... All the other `GenericXxxx` are in the global NS. But I can move GenericType under Moose. NP